core/modules/layout/css/grid-float.css

```diff
@@ -0,0 +1,958 @@
+/*!
+ * Bootstrap v4.0.0-alpha.2 (http://getbootstrap.com)
+ * Copyright 2011-2016 Twitter, Inc.
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+ */
+.container,
+.site-width-wrapper {
```

Comment:
These `.site-width-wrapper` classes don't seem be documented in Bootstrap and aren't used in our own templates. Did we add this but turn out not to need it?